cachito/workers/pkg_managers/gomod.py

```diff
@@ -152,7 +152,7 @@ def release(self) -> str:
         # lazy evaluation: defer running 'go'
         if not self._release:
             output = self(["version"])
-            log.info(f"Go release: {output}")
+            log.debug(f"Go release: {output}")
```

Comment:
In which case I'll just drop the commit instead of dropping the log since it provides 2 different kinds of information: what the bundled version reports and what go.mod reports and transitively what we'll use.

Reply:
Hmm, the commit was cherry-picked from cachi2 where I apparently mistakenly paired another log line in the commit which should have been placed in a standalone one. So what I'll do is I'll drop the log-level change and only keep the other log info on the actual go release used and reword the commit message.